testbed/build.sh

```diff
@@ -34,6 +34,7 @@ source $CONDA_ROOT/etc/profile.d/conda.sh
 conda activate base
 
 if [ ! -f $CONDA_ROOT/.created ]; then
+    conda config --prepend channels conda-forge --system
```

Comment:
This is to make sure we can get R environments on ARM-based macs.
Since these are just test environments, it's not going to change the functionality.